runtime/command/new/controller/view/show.gotext

```diff
@@ -19,7 +19,7 @@
 
 <br />
 
-<a href="..">Back</a>
```

Comment:
I swore this worked in my testing, but I guess not. `/post/1` would incorrectly resolve to `/` not `/post`.